src/views/manage-organizations.ejs

```diff
@@ -13,22 +13,30 @@
                                                   siret,
                                                   cached_libelle,
                                                   cached_adresse,
-                                                  cached_libelle_activite_principale
+                                                  cached_libelle_activite_principale,
                                               }) => { %>
                     <div class="fr-col-12 fr-col-lg-10">
-                        <div class="fr-tile fr-enlarge-link fr-tile--horizontal">
+                        <div class="fr-tile fr-tile--horizontal">
                             <div class="fr-tile__body">
-                                <h4 class="fr-tile__title">
-                                    <%= cached_libelle; %>
-                                </h4>
+                                <h3 class="fr-tile__title">
+                                    <span>
+                                    <a
+                                            class="fr-link fr-tile__link"
+                                            href="https://annuaire-entreprises.data.gouv.fr/etablissement/<%= siret; %>?mtm_campaign=moncomptepro"
+                                            target="_blank"
+                                            rel="noopener noreferrer"
+                                            aria-label="Détails sur l’Annuaire des Entreprises sur <%= cached_libelle; %>"
```

Comment:
Since the external link is now the main heading, I'd say it's now more important to focus on the org name now.

I'd probably do something like:

```suggestion
                                            aria-label="<%= cached_libelle; %> - détails sur l’Annuaire des Entreprises"
```

Reply:
And do the same for pending organizations below :)